src/hatti/ona/forms.cljs

```diff
@@ -193,6 +198,19 @@
                 (nil? thumb)          answer
                 :else                 [:a {:href image :target "_blank"}
                                        [:img {:width "80px" :src thumb}]]))
+       :vid (let [image (:download_url answer)
+                  thumb (or (:small_download_url answer) image)
+                  fname (last-url-param (:filename answer))]
+              (cond
+                (or (nil? answer)
+                    (string? answer)) answer
+                compact?              (format
```

Comment:
Given that this code is going to be moved back into the main UI for the API and given the time already spent on the issue I preferred not to refactor it (especially all the for loops in it) in the assumption that it would be of no value. It would be better to do it all at once when moving it to the main UI.